packages/shared/components/inputs/NetworkInput.svelte

```diff
@@ -1,48 +1,49 @@
 <script lang="ts">
     import { Modal, SelectorInput, IOption } from 'shared/components'
     import { activeProfile, getNetworkHrp } from '@core/profile'
-    import { NETWORK_ADDRESS, DestinationNetwork } from '@core/layer-2'
     import { validateBech32Address } from '@core/utils'
+    import { isIscpChain } from '@core/network'
+    import type { ChainConfiguration } from '@core/network'
 
-    const readonlyAttribute = $activeProfile?.isDeveloperProfile ? {} : { readonly: false }
-    const networkAddresses = NETWORK_ADDRESS[$activeProfile?.network?.id]
+    export let layer2ChainAddress: string | undefined = undefined
```

Comment:
is there a reason that you renamed this from  `networkAddress`? this var can both include a L1 and L2, no?

Reply:
I renamed it because we could get the layer2ChainAddress from the `SendFormPopup`, but you're right, inside `NetworkInput` it should be called differently.

Reply:
Not really, there is no L1 networkAddress. The network address (~alias address) only exists for ISCP Chains, so I'll change it to iscpChainAddress instead. If it's undefined then we have a L1 transaction. We need to figure out a way on how to handle this for EVM -> EVM transactions. But that's part of bigger refactor.